src/std/srfi/43.ss

```diff
@@ -32,16 +32,16 @@ package: std/srfi
   vector-any                      vector-every
 
   ;; * Mutators
-  vector-set!
+  #;vector-set!
   vector-swap!
-  vector-fill!
+  (rename-out vector-fill! (vector-fill! %%vector-fill!))
```

Comment:
you can use `(rename: vector-fill! %%vector-fill!)`, but if it's different than the builtin vector-fill! then we should export it and shadow the builtin (user can import from `<runtime>` if he needs it).